### PR TITLE
Add cron job to github_deploy server to prune docker

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Comment by iHiD** [comment]
@ErikSchierboom Have you added this manually? (terraform won't update the existing machine)

File: terraform/github_deploy/ami.sh [hunk 1]
```diff
@@ -13,6 +13,9 @@ docker run hello-world
 
 sudo systemctl enable docker
 
+echo "0 0 * * * docker image prune --force --filter until=\$(date +%s -d '1 week ago')" | sudo cron
+tab -u root -
```
Comment (dougal) [inline terraform/github_deploy/ami.sh:17]: Erroneous newline here?

Comment (ErikSchierboom) [reply terraform/github_deploy/ami.sh:17]: Yep. I'll fix

**Comment by ErikSchierboom** [comment]
> @ErikSchierboom Have you added this manually? (terraform won't update the existing machine)

I did. I'll revert

**Comment by ErikSchierboom** [comment]
I've undone what I did to the machine and fixed the newline

**Comment by iHiD** [comment]
@dougal Thanks!
@ErikSchierboom Can you add the correct command to the machine, please? (this script won't do anything by merging it - its just there for when I create new machines)

**Comment by ErikSchierboom** [comment]
Done

**Pull request merged**